src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandler.java

```diff
@@ -71,6 +79,29 @@ private void logInterchangeReceived(final Interchange interchange) {
         }
     }
 
+    private void sendRecep(final Interchange interchange) {
+        final String recepEdifact = recepProducerService.produceRecep(interchange);
+        final Interchange recep = edifactParser.parse(recepEdifact);
+        final var recepOutboundMessage = prepareRecepOutboundMessage(recepEdifact, recep);
+
+        meshOutboundQueueService.publish(recepOutboundMessage);
+
+        logRecepSentFor(interchange);
```

Comment:
In NHAIS there's just the logging for the recep (at INFO level anyway), but I liked the look of the log for the FHIR messages too. I kept them separate mostly because posting the recep could fail, and then we also wouldn't have that (imo helpful) log for the FHIR messages.

What do you think?

Reply:
There's no harm in keeping them separate for now, since they're related to different processes. We could always refactor later if it becomes too much